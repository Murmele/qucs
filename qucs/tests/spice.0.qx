load plugins/spice
load legacy
load legacy/components
load plugins/misc

<Components>
  <R R2 1 210 150 -26 15 0 0 "10 kOhm" 1 "26.85" 0 "0.0" 0 "0.0" 0 "26.85" 0 "european" 0>
  <R R5 1 520 150 -26 15 0 0 "10 kOhm" 1 "26.85" 0 "0.0" 0 "0.0" 0 "26.85" 0 "european" 0>
</Components>

netlist l=spice
quit
