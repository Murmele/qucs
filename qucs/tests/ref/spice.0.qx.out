
netlisting...
R2 ... stub
R5 ... stub
